File: HB/pysrc/프로그래머스/레벨0/Day24/a로b만들기.py
def solution(before, after):
    answer = 1
    b_=[]
    a_ =[]
    for i in before:
        b_.append(i)
    for i in after:
        a_.append(i)
    #if a_.sort() == b_.sort():
        #answer=1
    for i in range(len(a_)):
        if a_[i] != b_[i]:
            answer*=0
    return answer

File: SB/src/프로그래머스/레벨0/Day21/A로_B_만들기.java
package 프로그래머스.레벨0.Day21;
import java.util.*;
public class A로_B_만들기 {
    class Solution {
        public int solution(String before, String after) {
            char beforeCharArr[] = before.toCharArray();
            char afterCharArr[] = after.toCharArray();
            Arrays.sort(beforeCharArr);
            Arrays.sort(afterCharArr);
            before = String.valueOf(beforeCharArr);
            after = new String(afterCharArr);
            if (before.equals(after)) {
                return 1;
            } else {
                return 0;
            }

        }
    }
}
